e2e/volume_ops_test.go

```diff
@@ -112,27 +111,95 @@ func testVolumeCreate(t *testing.T) {
 	createReq.Name = "##@@#@!#@!!@#"
 	_, err = client.VolumeCreate(createReq)
 	r.NotNil(err)
+
 }
 
+func testVolumeCreateWithFlags(t *testing.T) {
+	r := require.New(t)
+
+	var brickPaths []string
+
+	for i := 1; i <= 4; i++ {
+		brickPaths = append(brickPaths, fmt.Sprintf(baseWorkdir+t.Name()+"/b/%d", i))
+	}
+
+	flags := make(map[string]bool)
+	//set flags to allow rootdir
+	flags["allow-root-dir"] = true
+	//set flags create brick dir
+	flags["create-brick-dir"] = true
+
+	createReqBrick := api.VolCreateReq{
+		Name: "createbrick",
```

Comment:
Name the volume after the test - use `t.Name()`

Reply:
done